daq_macros.py

```diff
@@ -1941,25 +1948,37 @@ def snakeRasterBluesky(rasterReqID, grain=""):
 
     else:  # yes, do row processing
       if daq_lib.abort_flag != 1:
+        print("processing rows")
         [thread.join(timeout=120) for thread in spotFindThreadList]
       else:
         logger.info("raster aborted, do not wait for spotfind threads")
-      logger.info(str(processedRasterRowCount) + "/" + str(rowCount))      
+      logger.info(str(processedRasterRowCount) + "/" + str(rowCount))
       rasterResult = generateGridMap(rasterRequest)
-  
+
       logger.info(f'protocol = {reqObj["protocol"]}')
       if (reqObj["protocol"] == "multiCol" or parentReqProtocol == "multiColQ"):
-        if (parentReqProtocol == "multiColQ"):    
+        if (parentReqProtocol == "multiColQ"):
           multiColThreshold  = parentReqObj["diffCutoff"]
         else:
-          multiColThreshold  = reqObj["diffCutoff"]         
-        gotoMaxRaster(rasterResult,multiColThreshold=multiColThreshold) 
+          multiColThreshold  = reqObj["diffCutoff"]
+        # gotoMaxRaster(rasterResult,multiColThreshold=multiColThreshold) 
```

Comment:
calls to other code can't just be removed. make a config variable or something that enables you to make NYX work with your code while AMX/FMX use the gotoMaxRaster() function